packages/api/src/utils/db-client.js

```diff
@@ -525,6 +525,18 @@ export class DBClient {
     /** @type {PostgrestQueryBuilder<definitions['auth_key']>} */
     const query = this.client.from('auth_key')
 
+    const { data: dataExists } = await query
```

Comment:
comment (blocking):
Let's get a test to check logic and behavior please!

Reply:
done!